### DISCUSSION TITLE
Unable to get into admin board of NPM

### DISCUSSION THREAD

**Discussion by GijsKruize** [NginxProxyManager/nginx-proxy-manager#3254] [General]
Recently I resetted my proxmox home server to fully redo everything properly. Today i re-installed Nginx Proxy Manager. but i am stuck in a loop if i want to acces port 443, my chromium browser sends a http request which is answered with an 400.

![Screenshot 2023-10-12 at 22 44 57](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/44362980/30ef73ca-5c53-4f58-9203-f2334ca37acd)

But if i force https, i get a ERR_SSL_UNRECOGNIZED_NAME_ALERT after reading some forms ,downgrading might help but i have tried multiple old and even older versions. But that does nothing, i used the standard docker compose script from the NPM site.

**Comment by lastsamurai26** [comment]
you need to type http://ip:81
